index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Quiz-Miniac</title>
+</head>
+<body>
+    <div class="container">
+        
+            <section class="scroll-item start">
+                <div class="title">
+                    <div class="highlight">
+                        <span class="Q">Q</span>
+                        <span class="U">U</span>
+                        <span class="I">I</span>
+                        <span class="Z">Z</span>
+                        <span class="M">M</span>
+                        <span class="I1">I</span>
+                        <span class="N">N</span>
+                        <span class="I2">I</span>
+                        <span class="A">A</span>
+                        <span class="C">C</span>
+                    </div>
+                </div>
+                <div class="get-down">
+                    <span>Scroll</span>
+                </div>
+
+            </section>
+            <section class="scroll-item mid">
+                <div class="category-head">
+                   <h3> Quiz-Miniac </h3>
+                 <a href="./Instruction/instruction.html">Rules</a>
+                </div>
+                <div class="category-cont">
+                    <div class="category-subhead">
+                        <h3>Categories</h3>
+                    </div>
+                    <div class="category-menu">
+                        <div class="menu-item">Anime</div>
+                        <div class="menu-item">English-Series</div>
+                        <div class="menu-item">Bollywood</div>
+                        <div class="menu-item">Hollywood</div>
+                    </div>
+                    <div class="category-item-cards">
+                        <div class="card-item">
+                            <div class="img-cont">
+                                <img src="https://www.imagedecor.com.au/wp-content/uploads/2018/04/PP32769_f.jpg" alt="" srcset="">
+                            </div>
```

Comment:
it's recommended to use alt attribute to help the  user incase the image does'nt load up

Reply:
Yes sure